javascript-objects/main.js

```diff
@@ -0,0 +1,32 @@
+const student = {
+  firstName: 'Aaron',
+  lastName: 'He',
+  age: 21,
+};
+const vehicle = {
+  make: 'Honda',
+  model: 'Accord',
+  year: 2013,
+};
+const pet = {
+  name: 'Rex',
+  type: 'Poodle',
+};
+
+const fullName = `${student.firstName} ${student.lastName}`;
+console.log('Value of fullName: ', fullName);
+
+student.livesInIrvine = false;
+student.previousOccupation = 'Frac Operator';
+console.log('Value of livesInIrvine: ', student.livesInIrvine);
+console.log('Value of previousOccupation: ', student.previousOccupation);
+
+vehicle.color = 'Silver';
+vehicle.isConvertible = false;
+console.log('Value of color: ', vehicle.color);
+console.log('Value of isConvertible: ', vehicle.isConvertible);
```

Comment:
These should be using bracket notation rather than dot notation

Reply:
<img width="412" alt="image" src="https://github.com/AaronHe09/c0523-code-solutions/assets/79122214/91505143-aae5-4f05-bae4-49fe7eaa5ac5">